modules/core/nuxeo-core-convert/src/test/java/org/nuxeo/ecm/core/convert/service/TestMimeTypeTranslationHelper.java

```diff
@@ -97,4 +97,13 @@ public void testHasCompatibleMimeType() {
         assertTrue(mimeTypeTranslationHelper.hasCompatibleMimeType(mimeTypes, "text/html"));
         assertTrue(mimeTypeTranslationHelper.hasCompatibleMimeType(mimeTypes, "application/pdf"));
     }
+
+    @Test
+    public void testGetConverterNamesWhenCOMimeTypeIsNull() {
+        //No NPE thrown
+        List<String> converterNames = mimeTypeTranslationHelper.getConverterNames("application/pdf", "text/plain");
+        assertEquals(2, converterNames.size());
+        assertTrue(converterNames.contains("toText1"));
+        assertTrue(converterNames.contains("toText2"));
+    }
```

Comment:
I'm wondering if this is necessary since `testGetConverterNames()` is already testing this exact code.
Simply adding the contribution below without the fix should make this test fail.